src/main.ts

```diff
@@ -20,6 +20,7 @@ const networkConfigUrl = (() => {
   }
 })();
 
+const walletParam = (networkConfigUrl !== "main" ? "wallet=main" : "");
```

Comment:
The url is the full url, you want `agoricNetName` above, you can just pull that outside the scope of the function so you can reference it here